src/SearchBar.js

```diff
@@ -1,5 +1,5 @@
 import React, { useState } from 'react';
-import {isMobile} from 'react-device-detect';
+// import {isMobile} from 'react-device-detect';  // commented out because this is now passed in via props.deviceIsMobile
```

Comment:
Sounds reasonable! 

I’d remove the line. One way to do PRs is to make these comments through the Github interface rather than leaving temporary comments in the code. By the current method, it’s hard to know what you intend to leave in and what’s just for readers of this particular PR

Reply:
I deleted the commented out line.